subsys/bluetooth/host/att.c

```diff
@@ -2918,6 +2926,33 @@ static struct bt_att_chan *att_chan_new(struct bt_att *att, atomic_val_t flags)
 	return chan;
 }
 
+#if defined(CONFIG_BT_EATT)
+#if defined(CONFIG_BT_TESTING)
+uint16_t bt_att_num_enhanced_bearers(struct bt_conn *conn)
```

Comment:
"Enhanced ATT bearer" is used in the spec, `eatt` and "channel" is used in zephyr. Can change to `bt_att_num_eatt` and keep that as a naming scheme in zephyr.

Reply:
`EATT` isn't really defined in the core spec in any case, so we won't see many references to it (although there are some). 

Zephyr does use `channel` instead of `bearer` as that is more fitting (at least in my opinion). 

I'd prefer to use `bt_att_num_eatt` to be more consistent with the Zephyr naming scheme.